audformat/core/database.py

```diff
@@ -558,6 +565,21 @@ def join_field(
                 else:
                     self[table_id] = table.copy()
 
+        # copy media files
+        if copy_media and self.root:
```

Comment:
I would check for `self.root is not None` here as you do later when you check for `other.root`.

Reply:
Changed it accordingly.